src/table/newgrf_debug_data.h

```diff
@@ -272,11 +272,38 @@ static const NIFeature _nif_industrytile = {
 /*** NewGRF industries ***/
 
 static const NIProperty _nip_industries[] = {
-	NIP(0x10, Industry, produced_cargo[0], NIT_CARGO, "produced cargo 0"),
-	NIP(0x10, Industry, produced_cargo[1], NIT_CARGO, "produced cargo 1"),
-	NIP(0x11, Industry, accepts_cargo[0],  NIT_CARGO, "accepted cargo 0"),
-	NIP(0x11, Industry, accepts_cargo[1],  NIT_CARGO, "accepted cargo 1"),
-	NIP(0x11, Industry, accepts_cargo[2],  NIT_CARGO, "accepted cargo 2"),
+	NIP(0x10, Industry, produced_cargo[ 0], NIT_CARGO, "produced cargo 0"),
```

Comment:
Should maybe point to properties 0x25 and 0x26 instead of the old ones here. Or maybe point to both property pairs? Or even more?

Reply:
I would go for the new ones, 0x25 and 0x26.